loopd/view.go

```diff
@@ -37,8 +38,16 @@ func view(config *Config, lisCfg *ListenerCfg) error {
 		chainParams,
 	)
 
+	var assetClient *assets.TapdClient
+	if config.Tapd.Host != "" {
+		assetClient, err = assets.NewTapdClient(config.Tapd)
+		if err != nil {
+			return err
+		}
+	}
+
 	swapClient, cleanup, err := getClient(
-		config, swapDb, sweeperDb, &lnd.LndServices,
+		config, swapDb, sweeperDb, &lnd.LndServices, assetClient,
```

Comment:
just checking, there's no special treatment of asset out swaps in `viewOut` required, right? Like the `FetchLoopOutSwaps` is handling all?